### DISCUSSION TITLE
BULK FILE

### DISCUSSION THREAD

**Discussion by news2023** [amzn/ads-advanced-tools-docs#72] [Q&A]
1.what are the list of  changes I can put under OPERATION in amazon ads  bulk file? aside from update, create, archive.
 
2.follow up question, under OPERATION - if I didn't put any changes like update, create, etc. - what will be the possible result or will there be a possible changes that may happen to may ads?

**Comment by DrRockett** [comment]
Hi! I believe this was answered [in this other discussion](https://github.com/amzn/ads-advanced-tools-docs/discussions/72), but just to close the loop: The "Operation" value can only be one of three options: Create, Update, or Archive. 

And if you don't put anything in that field, then no changes will occur. So if you have a downloaded bulk file with lots of rows, you can just make the updates to the rows you want to change, making sure you have the correct value in the "Operation" field for the rows you want to change. All of the other rows with no "Operation" value will remain unchanged.
